engine/util/timespan.cpp

```diff
@@ -3,15 +3,13 @@
 // Send questions to natehieter@gmail.com
 // ==========================================================================
 
-#include "sc_timespan.hpp"
-
 #include "fmt/format.h"
-#include "fmt/ostream.h"
+#include "sc_timespan.hpp"
```

Comment:
Leave the order as-is?

Reply:
that was clang-format, because the files have different names. Will fix.